src/utils.jl

```diff
@@ -1,27 +1,27 @@
-function isfuncexpr(ex, name=nothing)
-    checkname(fdef::Expr, name)            = checkname(fdef.args[1], name)
-    checkname(fname::Symbol, name::Symbol) = begin
-        fname === name && return true
-        startswith(string(name), string('#', fname, '#')) && return true
-        return false
+function checkname(fdef::Expr, name)
+    fdef.head === :where && return checkname(fdef.args[1], name)
+    if fdef.args[1] isa Expr
+        # E.g. `function Mod.bar.foo(a, b)`
```

Comment:
Should we check for a `.` head?

Reply:
Sure, but I don't know what to do in case it is not. Any examples of when that happens. I guess I just return `false` for now.